src/remote_cluster.rs

```diff
@@ -127,6 +127,11 @@ impl RemoteCluster {
         *active = Some(name);
     }
 
+    pub fn unset_active_scope(&self) {
```

Comment:
Should these new functions just be parameters to set_active_bucket and set_active_scope?

Reply:
Done!